python/run-tests.py

```diff
@@ -182,4 +188,7 @@ def run_delta_connect_codegen_python(root_dir):
     run_mypy_tests(root_dir)
     run_pypi_packaging_tests(root_dir)
     run_delta_connect_codegen_python(root_dir)
-    test(root_dir, package)
+    test(root_dir, "delta", package)
+    is_spark_master = os.getenv("IS_SPARK_MASTER")
```

Comment:
Seems like `IS_SPARK_MASTER` is only set when running the connect suites. Can we instead rename it something like `RUN_SPARK_CONNECT_TESTS`?

Reply:
@tomvanbussel Yeah I also thought about this 50 minutes ago haha